templates/account/logout.html

```diff
@@ -20,6 +20,14 @@
             {% element button type="submit" %}
                 {% trans 'Sign Out' %}
             {% endelement %}
+            {% element button type="button" id="cancel-button" %}
+            {% trans 'Cancel' %}
+            {% endelement %}
         {% endslot %}
     {% endelement %}
+    <script>
+        document.getElementById('cancel-button').addEventListener('click', function() {
+            history.back();
```

Comment:
using history.back() may be [unpredictable](https://stackoverflow.com/questions/25068936/unpredictable-results-on-google-chrome) sometimes, I think it is better to do it in a pythonic way.

Reply:
Yeah I agree with not using Javascript for this.